packages/leafygreen-provider/src/LeafyGreenProvider.tsx

```diff
@@ -1,13 +1,24 @@
 import React from 'react';
```

Comment:
Didn't catch this until late but this file and the other really should be `LeafyGreenContext` that exports the provider.

Reply:
👍